src/styles/actor/_coinage.scss

```diff
@@ -123,23 +123,19 @@
     justify-content: flex-end;
     padding: 0 0.25rem;
 
-    h3 {
+    .item-name {
+        color: var(--color-text-light-0);
```

Comment:
I'd prefer to shy away from appv1 variables if possible: how does `--color-light-1` fare here?

Reply:
Let me try some out, I didn't catch that it was using an old css variable. Its not quite the same but it looks fine to me:

![image](https://github.com/user-attachments/assets/d1a0af80-deff-4cff-9d45-b2e0546a1566)

Reply:
If we want to use whatever `--color-text-light-0` is, let's copy its value over to a system color variable

Reply:
This change actually looks fine. Its pure white vs slightly off white. Might even be considered an improvement.